doc/examples/color_exposure/plot_rgb_to_hsv.py

```diff
@@ -0,0 +1,44 @@
+"""
+================
+RGB to HSV
+================
+
+This example converts an image with RGB channels into an image with HSV (Hue,
+Saturation,Value) channels.
+
+This coordinate system is useful for color-based thresholding since the color
```

Comment:
`coordinate system` -> `representation` or smth similar.
I'd suggest to elaborate in this paragraph a bit more: say, for example, that in many cases objects in image can have distinct intensities or hues, therefore, HSV representation can be effectively used to perform simple segmentation tasks merely by thresholding.